test/packages/good/kibana/csp_rule_template/good-csp-rule-template-abc-1.json

```diff
@@ -14,7 +14,7 @@
       "Kubernetes",
       "Containers"
     ],
-    "benchmark": { "name": "CIS Kubernetes", "version": "1.4.1" },
+    "benchmark": [{ "name": "CIS Kubernetes V1.23", "id": "cis_k8s", "version": "v1.0.0" }],
```

Comment:
What is the context of this change? Is this template invalid? Should we add some validation to the spec?